### ISSUE TITLE
Cavestory works, but it's broken

### ISSUE THREAD

**Issue by Sayo4908** [madcock/sf2000_multicore_cores#4]
Hi, on Multicore 0.10, cavestory works, but all monsters aren't moving, and breaking the wall on cave 1 is impossible, so you can't beat first level

**Comment by madcock** [comment]
Yeah, unfortunately the Cavestory (https://docs.libretro.com/library/nxengine/) core has been broken since the beginning on SF2000. The game can be played using the Genesis port though, which works perfectly, but is just a console port so it doesnt' have the full features (or sound quality) of the original game.

Leaving open as someone may want to investigate and try to fix one day. Not sure if it will ever be able to run full speed but at least these weird gameplay bugs should be fixable since desktop retroarch doesn't have them for this core.
